ui/ducks/bridge/selectors.ts

```diff
@@ -515,7 +524,7 @@ export const getValidationErrors = createDeepEqualSelector(
       },
       // Shown after fetching quotes
       isInsufficientGasForQuote: (balance?: BigNumber) => {
-        if (balance && activeQuote && fromToken) {
+        if (balance && activeQuote && fromToken && fromTokenInputValue) {
```

Comment:
So we only want to calculate insufficient gas for quote only after the user has typed something in.